From e0da18901684e0380ce5453f6698fae7d670f3df Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:30:53 -0400
Subject: [alsa] lx_core: Remove useless #if 0 .. #endif

Message-id: <140972823573-275-patchset-jkysela@redhat.com>
Patchwork-id: 91143
O-Subject: [RHEL 7.1 PATCH 274/442] ALSA: lx_core: Remove useless #if 0 .. #endif
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The code contained in these sections are only dev_dbg calls, that are already
removed whenever DEBUG isn't defined.

Remove the redundant constructs.

Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 8e6320064c3350cd8610cd23d4ef5c6926e33e48)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/lx6464es/lx_core.c b/sound/pci/lx6464es/lx_core.c
index 0ad6358..7c07086 100644
--- a/sound/pci/lx6464es/lx_core.c
+++ b/sound/pci/lx6464es/lx_core.c
@@ -515,7 +515,6 @@ int lx_buffer_ask(struct lx6464es *chip, u32 pipe, int is_capture,
     *r_needed += 1;
   }
 
-#if 0
   dev_dbg(chip->card->dev,
    "CMD_08_ASK_BUFFERS: needed %d, freed %d\n",
        *r_needed, *r_freed);
@@ -526,7 +525,6 @@ int lx_buffer_ask(struct lx6464es *chip, u32 pipe, int is_capture,
          chip->rmh.stat[i],
          chip->rmh.stat[i] & MASK_DATA_SIZE);
   }
-#endif
  }
 
  spin_unlock_irqrestore(&chip->msg_lock, flags);
@@ -1191,7 +1189,6 @@ irqreturn_t lx_interrupt(int irq, void *dev_id)
  if (irqsrc & MASK_SYS_STATUS_CMD_DONE)
   goto exit;
 
-#if 0
  if (irqsrc & MASK_SYS_STATUS_EOBI)
   dev_dbg(chip->card->dev, "interrupt: EOBI\n");
 
@@ -1203,7 +1200,6 @@ irqreturn_t lx_interrupt(int irq, void *dev_id)
 
  if (irqsrc & MASK_SYS_STATUS_ORUN)
   dev_dbg(chip->card->dev, "interrupt: ORUN\n");
-#endif
 
  if (async_pending) {
   u64 notified_in_pipe_mask = 0;
@@ -1230,7 +1226,6 @@ irqreturn_t lx_interrupt(int irq, void *dev_id)
  }
 
  if (async_escmd) {
-#if 0
   /* backdoor for ethersound commands
    *
    * for now, we do not need this
@@ -1238,7 +1233,6 @@ irqreturn_t lx_interrupt(int irq, void *dev_id)
    * */
 
   dev_dbg(chip->card->dev, "interrupt requests escmd handling\n");
-#endif
  }
 
 exit:
-- 
1.7.1